gov_uk_dashboards/assets/mobile-nav.js

```diff
@@ -18,14 +18,30 @@ if (typeof attachEventToDash !== 'undefined') {
         mobileNav.style.display = isOpen ? 'none' : 'inline-block';
         isOpen = !isOpen;
     }, false)
+
+// Wait until the 'mobile-navigation-items' element is loaded
+var checkExist = setInterval(function() {
+    var element = document.getElementById('mobile-navigation-items');
+    if (element) {
+        clearInterval(checkExist);
+        var links = element.getElementsByTagName('a');
+        console.log(links)
+        attachEventsToLinks(links);
+    }
+}, 100);
+
+function attachEventsToLinks(links) {
+    for (var i = 0; i < links.length; i++) {
+        var link = links[i];
+        link.addEventListener('click', function() {
+            var mobileNav =document.getElementById('nav-section');
+            if (mobileNav) {
+                mobileNav.style.display = 'none';
+            }
```

Comment:
Great work getting this working! Good can just use the mobile-navigation-items id so don't need changes in LG. 
I think will need to add "isOpen = false;" after line 39 to update the isOpen state when a link is clicked